metafix/src/test/java/org/metafacture/metafix/MetafixLookupTest.java

```diff
@@ -424,10 +438,24 @@ public void shouldIgnoreOptionsOnSubsequentLookupInExternalFileMap() {
     }
 
     @Test
-    @MetafixToDo("Lookup with unknown internal map should not touch record")
     public void shouldNotLookupInUnknownInternalMap() {
-        assertMap(
-                LOOKUP + " 'testMap')"
+        MetafixTestHelpers.assertFix(streamReceiver, Arrays.asList(
+                LOOKUP + " '" + CSV_MAP + "', sep_char:'\t')"
```

Comment:
This is not an _internal_ map.

Reply:
Right, thanks, fixed in [6f455eb](https://github.com/metafacture/metafacture-fix/pull/170/commits/6f455ebedd2d848537036e7e05075accf14a4b64).